css/style.css

```
@@ -0,0 +1,903 @@
+  *,
+*::after,

```

Looks very strange... why do you need to use this selector with pseudo element?

---

Thahks!
left over from old works
Fixed it
 ```suggestion
 * {
      box-sizing: border-box;
  }
  ```

---

Thanks
reset took from old works
Fixed it
```suggestion
 * {
      box-sizing: border-box;
  }

```